dspace-api/src/main/java/org/dspace/eperson/SubscribeCLITool.java

```diff
@@ -26,8 +26,8 @@
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.PosixParser;
 import org.apache.commons.lang.StringUtils;
-import org.apache.log4j.Level;
-import org.apache.log4j.Logger;
+import org.apache.logging.log4j.Logger;
+// Replace with new log4j 2: import org.apache.logging.log4j.Level;
```

Comment:
Wondering if this should be removed or replaced?

Reply:
I think it's a leftover comment.